code-studio/src/js/assets.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  showAssetManager: require('./assets/show'),
+  listStore: require('./assets/assetListStore')
```

Comment:
Ideally we wouldn't expose the underlying `dashboard.assets.listStore` globally.  Instead I can add methods to preload assets and get the dropdown lists.